src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -122,13 +122,13 @@ public function getFormErrorsForJS(FormInterface $form)
                     $error->getMessageTemplate(),
                     $error->getMessagePluralization(),
                     $error->getMessageParameters(),
-                    'form_error'
+                    'validators'
                 );
             } else {
                 $errors[$formId][] = $translator->trans(
                     $error->getMessageTemplate(),
                     $error->getMessageParameters(),
-                    'form_error'
+                    'validators'
```

Comment:
That's weird. Where does this translation domain come from?

Reply:
Well not 100% sure but it seems to me that this is the domain for default constraint error messages from symfony.
![Capture d’écran 2022-08-25 à 17 58 54](https://user-images.githubusercontent.com/55691343/186713593-fb1c35bb-9a63-4bb3-9f9c-34834ef7d2e5.png)

Reply:
Huh. But this means that those wordings come from Symfony and aren't translatable through the back office interface 🤔

Reply:
Maybe not. I have yet very small knowledge about this bo translation feature. But symfony already handles it for most (all ?) languages =>
![Capture d’écran 2022-08-25 à 18 33 45](https://user-images.githubusercontent.com/55691343/186720752-3ee61c05-87da-47c0-83ac-2761231f953a.png)

Reply:
@eternoendless indeed, the native messages from Symfony are not translatable but Symfony catalogs already handle them. Do we really need to replace each Symfony internal wording with ours just to allow translation them via our interface?

If so it implies going through all the form constraints to override their native messages and domains. It's not impossible but it's a long job and not the scope of this PR I think.

Reply:
We should find a way to make them translatable, but on a different PR of course.